common/changes/@rushstack/heft/benpapp-FixPluginNameErrorTypo_2020-09-08-23-39.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft",
+      "comment": "fix typo in an error message to read that plugins must define a \"pluginName\" property, rather than the former \"displayName\" property",
```

Comment:
```suggestion
      "comment": "Fix a typo in an error message to read that plugins must define a \"pluginName\" property, rather than the former \"displayName\" property",
```